content/articles/10-ui-ux-mistakes/index.md

```diff
@@ -0,0 +1,94 @@
+
+Here are a few UI/UX mistakes that really annoy us, the users.
+
+### 1. Images
+Graphics play a major in passing information. Too large image sizes will make your product take longer to load. The right size is a balance between the size of the image and the amount of detail you want to show.
+
+For example, when listing users with their avatars side by side, the avatars should be small in size since loading large avatars will take a lot of time. Banner images on the other hand should be large in size, because they are not used in a lot of places and their detail is important.
+
+Something else that you should avoid when using images, is low-quality images. Low-quality images lack the sense of detail.
+
+To balance image quality and size, optimization comes in to play.
+
+Image optimization reduces the size of an image to the minimum necessary to display it. This is done by compressing the image, reducing the number of colors, and removing unnecessary information.
+
+When dealing with many images, you should consider using services like [Cloudinary](https://cloudinary.com/) to host, cache and optimize your images.
+
+### 2. Placing all the validation warnings in the same place
+Each form field should have its own validation warnings just below the input. This makes it easier to find which fields are throwing the errors.
+
+![validation warnings placement](form-validation.png)
+
+### 3. Showing the validation errors before the user has finished typing in a field
+Throwing the validation errors before the user has finished inputting that email or password is veeery annoying.
+
+You should wait for the user to finish typing in a field before showing the validation errors. The errors should be shown only after the user has clicked outside the field, but not after every keystroke.
+
+### 4. Design inconsistencies
+You should use consistent styling to your components. If you decide to use a border-radius of 20, stick to that all through.
+
+![design inconsistencies](inconsistent-design.png)
+
+Using too much colors in your design is also a bad idea. It makes it hard to keep related components together.
+
+You can get balanced color palletes from <https://coolors.co>
+
+If you have a multi-platform app, the appearance of the app on all the platforms should be consistent.
+
+### 5. Very wide paragraph
+Having a very wide paragrah that spans more than 2/3 the width of the screen makes it hard to read.
+
+Large paragraphs make it hard to follow a sentence to the next line. Paragraphs should take a maximum of 2/3 of the width of the screen or less. They should also be centered to help the users concentrate on what they are reading and to be able to follow the content line after line.
```

Comment:
There are different screen sizes in various devices. The statement "2/3 of the screen" is misleading. Please use another measurement unit to drive your point.